src/webgpu/shader/execution/expression/call/builtin/texture_utils.ts

```diff
@@ -718,6 +726,37 @@ function zeroValuePerTexelComponent(components: TexelComponent[]) {
   return out;
 }
 
+const kSamplerFns: Record<GPUCompareFunction, (ref: number, v: number) => boolean> = {
+  never: (ref: number, v: number) => false,
+  less: (ref: number, v: number) => ref < v,
+  equal: (ref: number, v: number) => ref === v,
+  'less-equal': (ref: number, v: number) => ref <= v,
+  greater: (ref: number, v: number) => ref > v,
+  'not-equal': (ref: number, v: number) => ref !== v,
+  'greater-equal': (ref: number, v: number) => ref >= v,
+  always: (ref: number, v: number) => true,
```

Comment:
nit. Is there a reason that some of these are in '' while other's aren't?

Reply:
because you can't have `-` in an identifier and lint requires you remove the '' if they're not needed.

Reply:
Ah... Thanks for clarifying~